core/src/net/sf/openrocket/rocketcomponent/FinSet.java

```diff
@@ -943,9 +943,15 @@ public Coordinate getFinFront() {
 	public Coordinate[] getFinPointsWithTab() {
 		final Coordinate[] finPoints = getFinPoints();
 		final Coordinate[] tabPoints = getTabPoints();
-		
+
 		Coordinate[] combinedPoints = Arrays.copyOf(finPoints, finPoints.length + tabPoints.length);
-		System.arraycopy(tabPoints, 0, combinedPoints, finPoints.length, tabPoints.length);
+
+		//System.arraycopy(tabPoints, 0, combinedPoints, finPoints.length, tabPoints.length);
+		combinedPoints[finPoints.length + 3] = tabPoints[0];
+		combinedPoints[finPoints.length + 2] = tabPoints[1];
+		combinedPoints[finPoints.length + 1] = tabPoints[2];
+		combinedPoints[finPoints.length + 0] = tabPoints[3];
```

Comment:
It turns out there's already a method which takes care of this within FinSet.java itself. This whole method can simply be replaced with:

```java
	public Coordinate[] getFinPointsWithTab() {
		return combineCurves(getFinPoints(), getTabPoints());
	}
```

Reply:
hahahaha, I wrote that function!  Guess I had a good reason for it :)